startMission

```diff
@@ -271,6 +271,10 @@ function cleanup()
   unset WHITE
   unset RESTORE
   unset DEAD
+  for objectScript in $inventoryDir
+  do
+    unset -f $objectScript
+  done
```

Comment:
The following code may ensure that the result of the unset command in the for loop is called as "unset -f funcName" instead of "unset -f /full/path/to/functionScript".  Functions set by the --init option due to thier existence in inventoryDir will need to match the name of the objectScript they are in
```
for objectScript in $inventoryDir
  do
    local objectScript=$(eval "printf '%q\n' "${objectScript##*/}"")
    unset -f $objectScript
  done
```

Resources:
printf %q ARGUMENT is printed in a format that can be reused as shell input, escaping non-print‐ able characters with the proposed POSIX $'' syntax

"${PWD##/}" is called parameter expansion on the current working directory and this is where I got my answer from. -> https://stackoverflow.com/questions/1371261/get-current-directory-name-without-full-path-in-a-bash-script

"${room##/}" is parameter expansion on the $room variable created by the For loop.  The parameter expansion in both cases should focus on everything after the last "/" in the file path, meaning that it will return only the filenames or directory names in a particular location, so this parameter expansion should work for you.

eval allows you to evaluate the printf statement and set the output to the local variable